src/lms-harmonizer/tests_integration_pgsql/procs/harmonize_sections/test_lmssections_canvas.py

```diff
@@ -16,113 +16,112 @@
 SOURCE_SYSTEM = "Canvas"
 
 
-def describe_when_harmonizing_canvas_sections():
```

Comment:
I changed these section tests back to my version, as they were throwing "ODBC escape convert error"